engine-tests/src/test/java/org/terasology/entitySystem/OwnershipHelperTest.java

```diff
@@ -67,6 +74,21 @@ public void testListsOwnedEntities() {
         EntityRef ownerEntity = entityManager.create(ownerComp);
 
         OwnershipHelper helper = new OwnershipHelper(entityManager.getComponentLibrary());
-        assertEqualsContent(Lists.newArrayList(ownedEntity), helper.listOwnedEntities(ownerEntity));
+        assertEqualsContent(helper.listOwnedEntities(ownerEntity), Lists.newArrayList(ownedEntity));
     }
+
+    //TODO: needs to be replaced with alternative
+    public static <T> void assertEqualsContent(Collection<? extends T> expected, Collection<? extends T> actual) {
```

Comment:
[assertIterableEquals](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertIterableEquals(java.lang.Iterable,java.lang.Iterable))

but for this specific case, you're looking at a list you expect to have exactly one element, so you could also write an assertion that it is of size 1 and that its element has the expected value.